### PR TITLE
Adds still missing German translations to PFE

### PR DESCRIPTION
**Pull request by AnLand**
Provides still missing translation to PFE and corrects some typos of the previous version.

Translation does not include any texts of legal importance (like User Agreement and Privacy Policy) and ignores Project Builder instructions. If any of these should be included, please let me know.

Staging branch URL: https://pr-{NUMBER}.pfe-preview.zooniverse.org

Fixes # .

Describe your changes.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/locales/de.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ export default {
     }
   },
   privacy: {
-    title: 'Zooniverse User Agreement and Privacy Policy',
+    title: 'Zooniverse User Agreement and Privacy Policy (Zooniverse Nutzungsvereinbarung und Datenschutzerklärung)',
```
Comment (srallen) [inline app/locales/de.js]: Is there a reason to leave the English here?

File: app/locales/de.js [hunk 2]
```diff
@@ -246,18 +246,18 @@ export default {
         about: 'Über',
         publications: 'Publikationen',
         ourTeam: 'Unser Team',
-        acknowledgements: 'Acknowledgements',
+        acknowledgements: 'Acknowledgements / Kreditierung',
```
Comment (srallen) [inline app/locales/de.js]: Also curious about why the English is left here.

**Comment by AnLand** [comment]
Hi! 

"Acknowledgements" is really hard to translate into German, especially with what it means in a research / academic context. If both terms are too much, I would go with "Acknowledgements" only as Zuzi did in the Czech translation. "Kreditierung" (another terms is just "Anerkennung" which basically means "recognition") is not specific enough.

For 'Zooniverse User Agreement and Privacy Policy (Zooniverse Nutzungsvereinbarung und Datenschutzerklärung)' I left the English in it, because the legal stuff is in English only. But I wanted people to know what it is about. Especially as these are linked several times on different pages. I can translate these regulations, too, no problem. But I am not a lawyer. 

Please let me know how you would like to have it. I just need some guidance.

Thanks!

**Comment by srallen** [comment]
@AnLand Thanks for the explanations. I just wanted to be sure the left in English wasn't a mistake. The reasons you gave make sense.

**Pull request merged**